src/encoded/search_views.py

```diff
@@ -0,0 +1,252 @@
+from pyramid.view import view_config
+
+from snovault.elasticsearch.searches.interfaces import AUDIT_TITLE
```

Comment:
This is not significant. Why are these (from snovault.elasticsearch.searches.interfaces) not group together?